MIAUI/UserRepository.cs

```diff
@@ -0,0 +1,74 @@
+﻿using MIAUI.Model;
+using SQLite;
+
+namespace MIAUI;
+public class UserRepository
+{
+    string _dbPath;
+    private SQLiteConnection conn;
+    public string StatusMessage { get; set; }
+
+    public UserRepository(string dbPath)
+    {
+        _dbPath = dbPath;
+    }
+
+    // connect to the database
+    private void Init()
+    {
+        if (conn is not null)
+            return;
+
+        conn = new SQLiteConnection(_dbPath);
+        conn.CreateTable<User>();
+    }
+
+    public void AddNewPerson(string name, string email)
+    {
+        int result = 0;
+        try
+        {
+            Init();
+
+            // basic validation to ensure a name was entered
+            if (string.IsNullOrEmpty(name))
+                throw new Exception("Valid name required");
+
+            result = conn.Insert(new User { Name = name, Email = email });
+
+            // TODO: Insert the new person into the database
```

Comment:
```suggestion
            // TODO: Insert the new person into the database
```
Do you need this todo still?